### DISCUSSION TITLE
Dashy ignoring theme setting?

### DISCUSSION THREAD

**Discussion by szinn** [Lissy93/dashy#773] [Help and Support]
Everything has been working fine until yesterday. When Dashy loads, it always chooses the default theme. Going into the config, it shows "Adventure-Basic" in the theme drop-down and if I choose that again, it switches to Adventure-Basic. Refresh the window and it's back to Default theme again.

View config has adventure-basic as the theme even though it is Adventure-Basic in the conf.yml file.

App info shows no errors and log file doesn't show anything either.

**Comment by lissy93** [comment]
Looks like it could be a bug, caused by #752
I will look into it...
In the meantime, you can just set `theme` under `appConfig`, and this should work as expected.

**Comment by szinn** [comment]
Thanks - I actually do have the theme set in the appConfig section and am using 2.1.0 which I don't think has that merged PR in it.

**Reply by szinn** [reply]
Interesting - I changed the setting in conf.yml to set theme to "adventure-basic" and it started working again. It did not like "Adventure-Basic" which is what shows in the popdown.
